cmd_config.go

```
@@ -0,0 +1,240 @@
+
+	// Create config command
+	createCmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create sample configuration file",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if _, err := os.Stat(configPath); err == nil {
+				fmt.Printf("Configuration file already exists: %s\n", configPath)
+				fmt.Print("Overwrite? (y/N): ")
+				
+				var response string
+				fmt.Scanln(&response)
```

golangci-lint (1.64.8)

Error return value of `fmt.Scanln` is not checked

(errcheck)